tests/Maker/MakeEntityTest.php

```diff
@@ -37,12 +37,26 @@ private function createMakeEntityTest(bool $withDatabase = true): MakerTestDetai
 
     private function createMakeEntityTestForMercure(): MakerTestDetails
     {
+        if (getenv('MAKER_SKIP_MERCURE_TEST')) {
+            // This test is skipped, don't worry about persistence
+            return $this->createMakerTest()
+                ->skipTest('MAKER_SKIP_MERCURE_TEST set to true')
+            ;
+        }
+
+        // @legacy - MakeEntity uses ux-turbo-mercure (archived), it needs to use ux-turbo (mercure built in) for Symfony 7.0
+        if ('7.0.x-dev' === $_SERVER['SYMFONY_VERSION']) {
+            return $this->createMakerTest()
+                ->skipTest('symfony/ux-turbo-mercure is not supported on Symfony 7.')
```

Comment:
Chicken or the egg... `MakeEntity` needs to be refactored to use `ux-turbo` and at first glance doesn't appear to be a "1 for 1" swap with `ux-turbo-mercure`. To keep the entire test suite from blowing up due to a deprecation - this skip conditional was moved here for now.

Reply:
Ok for now for me